src/compas_timber/connections/x_halflap.py

```diff
@@ -1,45 +1,174 @@
-from compas_timber.elements import MillVolume
+from compas.tolerance import TOL
+
+from compas_timber._fabrication import Lap
+from compas_timber.connections.utilities import beam_ref_side_incidence
+from compas_timber.connections.utilities import beam_ref_side_incidence_with_vector
```

Comment:
these should be importable from the 2nd level i.e. `from compas_timber.connections import ...` or relative import like the ones below could also work since we're in the  same package anyways